### ISSUE TITLE
Chemical Treats Rattus Norvegicus 

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#707]
This is a question of answer quality - the overall result is not in question but the supporting evidence in the supporting paths which have Rattus Norvegicus seem fairly general 

UI Test > MVP1 > Common Cold:  https://ui.test.transltr.io/main/results?l=Common%20Cold&i=MONDO:0005709&t=0&r=0&q=a49c78af-b108-490e-b48f-f7aa7a5e568c

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/03ce46eb-e084-4010-9923-df1202bcda91)

Epoprostenol Treats Rattus Norvegicus
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/69ab73b5-42f3-44cd-95ef-bd43b6dd8992)

Rattus Norvegicus Has Part Ins
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/7faa1545-7cb2-44a9-b32a-770479d1bd04)

Ins Associated With Common Cold
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/1bff4b55-ec3f-4272-b89f-3333b7fac687)

Looks to be coming from ARAX - ARAX GUI:  https://arax.ncats.io/beta/?r=a49c78af-b108-490e-b48f-f7aa7a5e568c
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/4ba349a1-dd63-406c-9775-7f98228eadd0)

**Comment by gglusman** [comment]
Interesting that the first publication allegedly supporting the 'RN has_part INS' edge is... in mice, and rats aren't mentioned in it at all.

**Comment by sstemann** [comment]
@edeutsch this result in question is from ARAX via semmed, can you take a look?

**Comment by edeutsch** [comment]
yeah, it's coming from the xDTD machine learning model that was trained with data that includes a silly assertion from SemMedDB. On the plus side, there do seem to be ample publications that indicate that many researchers do indeed treat rats with Epoprostenol.

**Comment by bill-baumgartner** [comment]
One potential solution to this issue could be to leverage the domain and range of Biolink predicates. In the case of `biolink:treats` [its range is specified as](https://github.com/biolink/biolink-model/blob/0b39cabb2d01b400653ee0f71ea72babd2be7c82/biolink-model.yaml#L3481) `disease or phenotypic feature`. Assertions like `chemical - treats - taxon` are, therefore, technically not compliant with the Biolink model and could be filtered automatically.
